src/modal.js

```diff
@@ -0,0 +1,59 @@
+import extend from "./util.js";
+import Base from "./base.js";
+
+function Modal(elt, config, onSubmit) {
+  Modal.superclass.constructor.call(this, elt);
+  this.config = config;
+  this.onSubmit = onSubmit;
+}
+extend(Modal, Base);
+
+Modal.prototype.render = function (){
+  const modalConf = this.elt.querySelector('#main');
```

Comment:
#id is a unique identifier for the whole page, thus you won't be able to create another modal screen in the application. Please, replace it with class